WORKSPACE

```diff
@@ -25,7 +25,6 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 
 # Keep this version in sync with:
 #  * The BAZEL environment variable defined in .github/workflows/ci.yml.
-#  * The Bazel version installed for nightly builds (Googlers, see go/tb-tomservo).
```

Comment:
Maybe we could keep the first half of the comment `The Bazel version installed for nightly builds.`?

Reply:
The comment above it (referencing .github/workflows/ci.yml) is the version installed for nightly builds (since the nightly workflow invokes the CI job). How about:

`The Bazel environment version defined in .github/workflows/ci.yml, which is used for CI and nightly builds.`?

Reply:
Makes sense!